whats-new-in-tidb-4.0.md

```diff
@@ -1,92 +0,0 @@
----
-title: What's New in TiDB 4.0
```

Comment:
I think this file can be safely deleted after adding an aliases to https://docs.pingcap.com/tidb/v4.0/whats-new-in-tidb-4.0.

Reply:
@TomShawn Yes. Would you please help with adding the `aliases` to that file?

Reply:
OK